src/components/Movies.js

```diff
@@ -0,0 +1,59 @@
+import "../App.css";
+import React from "react";
+
+
+class Movies extends React.Component {
+  constructor(){
+    super()
+
+    this.state ={
+      films:[],
+      currentFilms: null,
+    };
+  }
```

Comment:
```suggestion
    this.state ={
      films:[],
      currentFilm: null,
    };
  }
```
`this.state.currentFilm` represents **ONE** singular movie. You're meant to display **one** movie at a time, and that's what currentFilm should represent.